Survey/Sources/Presentation/Modules/Login/LoginView.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  LoginView.swift
+//  Survey
+//
+//  Created by Khanh on 27/07/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Combine
+import SwiftUI
+import UIKit
+
+struct LoginView: View {
+
+    private var input: LoginViewModel.Input
+    @ObservedObject var output: LoginViewModel.Output
+
+    private let loadTrigger = PassthroughSubject<Void, Never>()
+
+    @State private var image: UIImage? = UIImage(named: "ic_background")
+    @State private var email = ""
+    @State private var password = ""
+
+    private var loginTitle = "Login"
+
+    private let gradient = LinearGradient(
+        gradient: Gradient(stops: [
+            .init(color: .black, location: 0.0),
+            .init(color: .black.opacity(0.2), location: 1.0)
+        ]),
+        startPoint: .bottom,
+        endPoint: .top
+    )
+
+    var body: some View {
+        GeometryReader { geo in
+            ZStack {
+                backgroundSetup()
+                VStack {
+                    logoSetup()
+                        .frame(height: geo.size.height * (1.0 / 3.0))
+                    componentSetup()
+                        .frame(height: geo.size.height * (1.0 / 3.0))
+                    Text("")
```

Comment:
Not sure what is this for since it's empty 🤔 If you want to show space, maybe use Spacer() or empty VStack

Reply:
I divided the screen into 3 parts and at a time I didn't think of `Spacer()`, so `Text("")` was like a empty stack view 🥲 . I fixed it using `Spacer()` at https://github.com/nkhanh44/ic-survey-ios/pull/13/commits/45ae841b35aa065645f32fb9c3e9b222f1e64540